src/Backend/Core/Engine/Csv.php

```diff
@@ -63,7 +63,7 @@ public static function outputCSV(
 
     private static function getLineEnding(): string
     {
-        $lineEnding = Authentication::getUser()->getSetting('csv_line_ending');
```

Comment:
Can you fix this in the installer instead? That would be the correct place for this.